acestep/inference_seed_fallback_test.py

```
@@ -0,0 +1,109 @@
+
+
+class SingularSeedFallbackTests(unittest.TestCase):
+    """``params.seed`` must reach the DiT when ``config.seeds`` is unset."""
+
+    def test_bare_seed_reaches_dit_with_random_seed_disabled(self):
+        """config.seeds=None + params.seed=42 + use_random_seed=False must forward seed '42'."""
+        dit_handler = RecordingDitHandler()
+        llm_handler = _make_llm_handler()
+        params = GenerationParams(
+            task_type="complete",  # direct-conditioning task: skips the LM, keeps this test focused on seeds
+            src_audio="/tmp/source.wav",
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/source.wav"

(S108)